added a getter for the mMenuSize property

josomers opened the pull request:
For some very important functionality in my application I added a getter (as there is also a setter), getting the 'menu size'. I thought this could be useful adding to your lovely android-menudrawer library.

Commit:
added a getter for the mMenuSize property

JakeWharton commented on library/src/net/simonvt/menudrawer/MenuDrawer.java:
```diff
@@ -957,6 +957,10 @@ void saveState(Bundle state) {
     }
+    
+    public int getMenuSize() {
+    	return mMenuSize;
```
Needs to be indented with four spaces.

SimonVT commented:
Needs to be moved up to the setter.

Commit:
implemented code review comments: moved code above setter and corrected indentation

SimonVT commented:
Sorry, didn't see you made the changes. I just added a setter on dev. Thanks though!

josomers commented:
No problem. I fixed the indentation and the position of the getter in my latest commit. I checked your code on dev and it's exactly what I added in my commit, but I guess you added the getter and not a setter (typo in your comment) ?

SimonVT commented:
There's both a getter and a setter on dev

josomers commented:
Yes, I know, but the setter already existed before. Thank you for adding this part into your code.
